### ISSUE TITLE
improve grammar-lex models

### ISSUE THREAD

**Issue by mikabr** [langcog/wordbank#88]

**Comment by mikabr** [comment]
First step from meeting today: fit quadratics (and possibly cubics), compare to logits.

**Comment by mikabr** [comment]
Dan and I fit some more models that look better. Our method is to fit an rlm for each language and measure with the formula:
```
score ~ I(vocab.mean^3) * age + I(vocab.mean^2) * age + vocab.mean * age - 1
```
Use that model to predict `score` for every combination of each age and the range of `vocab.mean` values (every 0.01 increment from 0 to 1). Then to bin age, average the predicted values for the ages in a given bin, weighted by the number of kids of that age.

Seems to look quite good for Norwegian and English, a little wonky for Danish, and have sparsity issues (as always) with Spanish.

![new_models](https://cloud.githubusercontent.com/assets/1414971/8224279/5d6b3392-1539-11e5-81bb-7c40d3fefb4d.png)

**Comment by mcfrank** [comment]
Those look great. We might consider checking whether the cubics are warranted for Spanish, given data sparsity. 

I like the idea of fitting only one model with continuous age and then deriving the fan based on predictions from that model. 

Mike

**Comment by mikabr** [comment]
Model comparison:
![model comparison](https://cloud.githubusercontent.com/assets/1414971/8265839/fdf2df4e-16c0-11e5-87f8-3a40b0eca520.png)

**Comment by mikabr** [comment]
Bootstrapped estimates of area differences, with 4 age groups:
![area diffs 4](https://cloud.githubusercontent.com/assets/1414971/8265845/1e633d82-16c1-11e5-8552-7199ed95214f.png)

with 2 age groups:
![age diffs 2](https://cloud.githubusercontent.com/assets/1414971/8265846/267d8928-16c1-11e5-8f4e-4ac384847e43.png)

**Comment by mcfrank** [comment]
Cool. Those look extremely consistent across languages!

Note that the four group split doesn't tell you anything extra because the age effects are all derived from the same model. That's why the bars are identical. 

Also - model comparison numbers at this level aren't telling us much, I think. Better to eyeball all the different fits for Spanish (maybe faceted by age group as well so you can see better) and then see if you think one functional form is doing something weirder or another is more reasonable. We could put numbers to this with much fancier models - eg bayes with some kind of non-crossing prior - but this kind of hand-examination is probably more useful for intuition, at least in the short term. 

Nice work!

Mike
